### PR TITLE
Add manual Up Next action to tvOS player

### PR DESCRIPTION
**Pull request by Quick104**
## Summary

- add an Up Next button to the tvOS transport when Next Up content is available or loading
- route manual presentation through the same existing postroll flow used by the automatic trigger
- share the availability predicate between the manual control and automatic timing path

## Validation

- `xcodebuild build -project Silo.xcodeproj -scheme SiloTV -destination "platform=tvOS Simulator,id=725C6034-1B19-45BE-BC94-46DDDEA14B03" CODE_SIGNING_ALLOWED=NO`
- launched the development build on tvOS 26.4 and played episodic content
- focused and activated the new transport control, confirming it opens the existing Next Up screen unchanged
- `git diff --check`

## Summary by CodeRabbit

* **New Features**
  * Added a “Show Up Next” control to the tvOS player transport controls when upcoming content is available.
  * Improved focus support for navigating to the Up Next action.

* **Bug Fixes**
  * Prevented the Up Next screen from opening unless the Up Next UI is actually available.
  * Refined Up Next completion/progress handling using a centralized policy (including prompt-window and end-of-file edge cases).

* **Tests**
  * Added unit tests covering the Up Next completion/progress policy behavior.

### CODE REVIEW

File: iosApp/iosApp/Screens/Player/PlayerViewModel.swift [hunk 1]
```diff
@@ -1666,10 +1674,12 @@ class PlayerViewModel {
...
-        return nextUpEpisode != nil
-            || !nextUpCarouselItems.isEmpty
-            || isLoadingNextUpEpisode
-            || isLoadingNextUpOnDeck
+        return canShowNextUpScreen
+    }
+
+    func showNextUpNow() {
+        guard canShowNextUpScreen else { return }
+        beginNextUpPostroll(videoEnded: false)
```
[the remark this answers is not in the document]
Comment (Quick104) [reply iosApp/iosApp/Screens/Player/PlayerViewModel.swift:1676]: Fixed in 5445656. Opening Up Next manually before the normal end window now preserves the current playback position when Play Now or an On Deck item is selected. Completion is only finalized at the normal prompt window or after EOF, and the offline cleanup path follows the same policy. Added four focused regression tests covering early manual presentation, the normal prompt window, EOF, and a hidden prompt; all pass, and the SiloTV simulator build succeeds.
